assets/js/public/controllers/statisticController.js

```diff
@@ -12,6 +12,114 @@ angular.module('zoomableApp').controller('statisticController', function($scope,
   // CHART VARIABLES
   $scope.series = ['Views'];                                // default series to show for graph
 
+  // HEATMAP VARIABLES
+  var sessions = {};
+  var compiledSessions = {};
+  var videoTotalTime = 0;
+  var totalCanvas = [];
+  var video = new Whammy.Video(1);
+
+  servicesAPI.getHeatMapStats(152).success(function (data) {
+    // mock view sessions data
+    var mockData = [
+      {coordinates: [0,0], videoTime: 0, videoTotalTime: 5},
+      {coordinates: [100,100], videoTime: 0, videoTotalTime: 5},
+      {coordinates: [200,200], videoTime: 1, videoTotalTime: 5},
+      {coordinates: [10,10], videoTime: 1, videoTotalTime: 5},
+      {coordinates: [10,50], videoTime: 1, videoTotalTime: 5},
+      {coordinates: [10,80], videoTime: 1, videoTotalTime: 5},
+      {coordinates: [250,100], videoTime: 2, videoTotalTime: 5},
+      {coordinates: [430,270], videoTime: 5, videoTotalTime: 5}
+    ];
+    sessions = mockData;
```

Comment:
Not changing this to mockData yet. Please merge PR #118 first to fix statistics data not sending correctly. Then we try if heatmap is working...